identity.go

```diff
@@ -77,10 +77,10 @@ func newProvisionalIdentity(config config, email string) (provisionalIdentity, e
 		return provisionalIdentity{}, err
 	}
 
-	provisionalIdentity := provisionalIdentity{
+	pid := provisionalIdentity{
```

Comment:
Beware that you're sometimes using pid for "public identity" and sometimes for "provisional identity". This could become confusing very fast.